web/crux/src/app/team/team.service.ts

```diff
@@ -319,17 +329,41 @@ export default class TeamService {
       throw new MailServiceException()
     }
 
+    const invite = await this.prisma.userInvitation.create({
+      data: {
+        userId: user.id,
+        teamId: team.id,
+        email: request.email,
+      },
+    })
+
     await this.notificationService.sendNotification({
       identityId: request.accessedBy,
       messageType: 'invite',
       message: { subject: request.email, team: team.name } as InviteMessage,
     })
 
-    const invite = await this.prisma.userInvitation.create({
+    return CreateEntityResponse.fromJSON({
+      id: invite.userId,
+      createdAt: invite.createdAt,
+    })
+  }
+
+  async reinviteUserToTeam(request: ReinviteUserRequest): Promise<CreateEntityResponse> {
+    const invite = await this.prisma.userInvitation.update({
+      where: {
+        userId_teamId: {
+          userId: request.userId,
+          teamId: request.id,
+        },
+      },
       data: {
-        userId: user.id,
-        teamId: team.id,
-        email: request.email,
+        status: 'pending',
+        createdAt: new Date(),
```

Comment:
If you update the `createdAt` then it will be invalid. You should add `updatedAt()` and update this field.